test/input.txt

```diff
@@ -148,6 +148,20 @@
   clear
   list
 
+##########################################################
+# test sort command
+##########################################################
+
+  # should add correctly and list non private information
```

Comment:
Good job in adding a test case for the IO, maybe you can start trying out the JUnit test suite to familiarise yourself with it.